### ISSUE TITLE
[Meizu 18] No VoLTE (call/SMS over LTE)

### ISSUE THREAD

**Issue by AndyCGYan** [phhusson/treble_experimentations#1894]
Device: Meizu 18 (meizu_18_CN)
PHH AOSP version: v309
Vendor version: R (Flyme 9)

On China Telecom (now a pure LTE carrier), with IMS APK installed & rebooted twice & IMS force toggles enabled, LTE works, but calls return "mobile network not available", and SMS are not sent.

Log of attempting a call here: [ct.txt](https://github.com/phhusson/treble_experimentations/files/6634664/ct.txt)

**Comment by phhusson** [comment]
Same carrier, on another phone, VoLTE works?

**Comment by AndyCGYan** [comment]
Yes, this SIM is one of my two dailies on a non-GSI phone.

**Comment by phhusson** [comment]
sorry, I meant another phone running same GSI

**Comment by AndyCGYan** [comment]
Yes also - this is actually my first time trying VoLTE on a GSI so I was uncertain, but the first other device I tried, Nubia Mini 5G, got it instantly working.

**Comment by miuma1** [comment]
Hello brother, where can I download ims apk?

**Comment by AndyCGYan** [comment]
@miuma1 For QCOM 64-bit: https://treble.phh.me/stable/ims-q.64.apk
